script/firebaseScriptLogWork.js

```diff
@@ -84,14 +84,15 @@ firebase.auth.EmailAuthProvider.PROVIDER_ID
 // tosUrl: '<your-tos-url>',
 // // Privacy policy url.
 // privacyPolicyUrl: '<your-privacy-policy-url>'
-};  
+};
 
 function createNewTask(title,des,priority,difficulty,creator,assignee,estimatedTime,status,category,creationDate) {
 
   var taskIdNumber = getIdNumber();
   var taskId = category[0]+taskIdNumber;
   var loggedWorkTotalTime = 0;
   var workDone = 0;
+  var completedDate = "Not Yet Completed";
```

Comment:
Please keep the default value of completedDate 0, and show the option of completed date only when it is not 0

Reply:
k anna  , i will do it .